cdap-ui/app/directives/complex-schema/complex-schema.js

```diff
@@ -94,12 +95,18 @@ function ComplexSchemaController (avsc, SCHEMA_TYPES, $scope, uuid, $timeout) {
       };
     });
 
+    if (!vm.isDisabled && vm.parsedSchema.length === 0) {
+      vm.addField();
```

Comment:
We seem to be doing `vm.formatOutput` inside `vm.addField`. Is it done again in line 102 because of the `$timeout`?